app/src/main/java/io/appium/uiautomator2/handler/ActiveElement.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.appium.uiautomator2.handler;
+
+import androidx.test.uiautomator.By;
+import androidx.test.uiautomator.UiObjectNotFoundException;
+
+import io.appium.uiautomator2.handler.request.SafeRequestHandler;
+import io.appium.uiautomator2.http.AppiumResponse;
+import io.appium.uiautomator2.http.IHttpRequest;
+import io.appium.uiautomator2.model.AndroidElement;
+import io.appium.uiautomator2.model.AppiumUIA2Driver;
+import io.appium.uiautomator2.model.Session;
+import io.appium.uiautomator2.model.internal.CustomUiDevice;
+import io.appium.uiautomator2.utils.Logger;
+
+import static io.appium.uiautomator2.utils.AXWindowHelpers.refreshAccessibilityCache;
+
+public class ActiveElement extends SafeRequestHandler {
+
+    public ActiveElement(String mappedUri) {
+        super(mappedUri);
+    }
+
+    @Override
+    protected AppiumResponse safeHandle(IHttpRequest request) throws UiObjectNotFoundException {
+        refreshAccessibilityCache();
+        Object element = CustomUiDevice.getInstance().findObject(By.focused(true));
```

Comment:
Please update the type according to the changes made in https://github.com/appium/appium-uiautomator2-server/pull/422

Reply:
Please add handler for the case when no element is found

Reply:
It's done

Reply:
Great suggestion @mykola-mokhnach -- thanks for the quick turn-around on the fix @vane